pylib/cqlshlib/test/test_cqlsh_completion.py

```diff
@@ -1205,18 +1211,45 @@ def test_complete_in_alter_role(self):
         self.trycompletions('ALTER ROLE foo WITH ACCESS TO ', choices=['ALL', 'DATACENTERS'])
         self.trycompletions('ALTER ROLE foo WITH ACCESS FROM ', choices=['ALL', 'CIDRS'])
 
-    def test_complete_in_create_user(self):
-        self.trycompletions('CREATE USER ', choices=['<username>', 'IF'])
-        self.trycompletions('CREATE USER IF ', immediate='NOT EXISTS ')
 
     def test_complete_in_drop_role(self):
         self.trycompletions('DROP ROLE ', choices=['<identifier>', 'IF', '<quotedName>'])
 
+
+    # IDENTITY checks
+    def test_complete_in_add_identity(self):
+        self.trycompletions('ADD ID', immediate='ENTITY ')
+        self.trycompletions('ADD IDENTITY IF ', immediate='NOT EXISTS ')
+        self.trycompletions('ADD IDENTITY IF NOT ', immediate='EXISTS ')
+        self.trycompletions('ADD IDENTITY ',
+            choices=['<pgStringLiteral>', '<quotedStringLiteral>', 'IF'])
+        self.trycompletions("ADD IDENTITY 'alice@example.com' TO R", immediate='OLE ')
+        self.trycompletions("ADD IDENTITY 'alice@example.com' ", immediate='TO ROLE ')
+        self.trycompletions("ADD IDENTITY 'alice@example.com' TO ROLE ",
+                            choices=['<identifier>', '<quotedName>'], other_choices_ok=True)
+        self.trycompletions("ADD IDENTITY 'alice@example.com' TO ROLE data_engineer ",
+                                    choices=[';'])
+        self.trycompletions("ADD IDENTITY IF NOT EXISTS 'alice@example.com' TO ROLE data_engineer ",
+                            choices=[';'])
+
+    def test_complete_in_drop_identity(self):
+        self.trycompletions('DROP IDENTITY ',
```

Comment:
Can we try autocompletion for:

- `D`
- `DR`
- `DROP I`
- `DROP ID`
- `DROP IDENTITY IF `
- ...